website/screens/components/dialog/usage/DialogUsagePage.tsx

```diff
@@ -36,6 +36,11 @@ const sections = [
           Modal dialog boxes should overlay only a portion of the underlying
           page to keep the user oriented within the workflow.
         </DxcBulletedList.Item>
+        <DxcBulletedList.Item>
+          In a dialog, the focus should remain within the modal box until the
+          user completes the required action (if any) or closes it. In addition,
+          the focus should appear on the first focusable child when it opens.
```

Comment:
This is not always the case since there are situations in which the interactions are limited and it may be advisable to set focus to the element that is likely to be most frequently used.

I think this can be reworded to generally the focus should appear on the first focusable child.